### PR TITLE
Sno-115 Tests

### PR DESCRIPTION
**Pull request by willronchetti**
- Add some tests to test new edge ngram behavior around edges
- DO NOT MERGE until snovault release is made

### CODE REVIEW

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4756039&bid=15862061)

File: src/encoded/tests/test_search.py [hunk 1]
```diff
@@ -137,6 +137,32 @@ def test_search_with_simple_query(workbook, testapp):
     assert not set(mouse_uuids).issubset(set(mauxz_uuids))
 
 
+def test_search_ngram(workbook, testapp):
```
Comment (j1z0) [inline src/encoded/tests/test_search.py]: Codacy Issue found: [redefinition of unused 'workbook' from line 2](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4756039)
